core/commands/keyencode/keyencode.go

```diff
@@ -1,11 +1,14 @@
 package keyencode
 
 import (
+	cmds "github.com/ipfs/go-ipfs-cmds"
 	peer "github.com/libp2p/go-libp2p-core/peer"
 	mbase "github.com/multiformats/go-multibase"
 )
 
-const IPNSKeyFormatOptionName = "ipns-base"
+const ipnsKeyFormatOptionName = "ipns-base"
+
+var OptionIPNSBase = cmds.StringOption(ipnsKeyFormatOptionName, "Encoding used for keys: Can either be a multibase encoded CID or a base58btc encoded multihash. Takes {b58mh|base36|k|base32|b...}.").WithDefault("base36")
```

Comment:
@Stebalien Any reason why reusing the same option repeatedly is not recommended?

Reply:
No, we do that for some shared options. It should be fine.